ls1intum/Apollon_standalone
# `Development`: Fix pdf export for communication diagrams

### Checklist

### Motivation and Context
This PR fixes arrows not appearing correctly in the communication link messages on pdf export


### Steps for Testing
#### Test Server
- Goto [test server](https://test1.apollon.ase.in.tum.de/)
- Create a new communication diagram
- Create multiple objects with links between them
- Place objects to see each arrow type (up, down, right, left)
- Observe that the arrows are correctly displayed on pdf export.


### Screenshots
#### Before
![before](https://user-images.githubusercontent.com/36473858/205502140-7bcbde98-1d9a-4873-8643-6a589c01bbf3.png)


#### After
![after](https://user-images.githubusercontent.com/36473858/205502146-c9485415-cc3a-4ef5-9c74-36749b7893b7.png)
